lib/api.js

```diff
@@ -766,6 +766,36 @@ module.exports = function(self, options) {
     return callback(null, result);
   };
 
+  // Retrieve all documents where the draft version has been trashed, but the
+  // live version has not. This method will call `self.getEditable` on the found
+  // docs, which means that the callback is called with the same signature
+  // `(null, modified, unmodified, committable)`
+
+  self.getTrashed = function(req, callback) {
+    return self.apos.docs.find(req, {}, { _id: 1, workflowGuid: 1 }).trash(true).published(null).toArray(function(err, draftDocs) {
+      if (err) {
+        return callback(err);
+      }
+
+      var workflowGuids = _.pluck(draftDocs, 'workflowGuid');
+
+      return self.apos.docs.find(req, {
+        workflowGuid: { $in: workflowGuids },
+        // We avoid calling the workflowLocale cursor filter, because that
+        // method will include doc types that are excluded from
+        // apostrophe-workflow
+        workflowLocale: self.liveify(req.locale)
+      }, { _id: 1 }).published(null).workflowLocale(null).toArray(function(err, liveDocs) {
```

Comment:
You need to include `.permission('edit')` in this chain so that we don't get back live docs we lack rights to commit.

Reply:
Pushed a fix for this!